Features

andynarf opened the pull request:
ready for code review.

mcrd25 commented on dist/index.html:
```diff
@@ -0,0 +1,12 @@
+    <title>Space Burgers</title>
+  </head>
+  <body>
+     <div id="menuid">
```
Ideally, the project instructions said to name it `content` which has more semantic meaning given the tabbed content requirement.
See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)

mcrd25 commented on src/menu.js:
```diff
@@ -0,0 +1,81 @@
+const menu = () => {
```
This is a cool shortcut to get a project, but does not really test your knowledge of JavaScript and Using JS modules. You are actually required to have tabbed content (Main, Menu and Contact (you are missing contact) )where each content is stored in its own module and exported. So Main would be its own module, menu and contact the same.

See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)

mcrd25 commented on webpack.config.js:
```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'bundle.js',
```
While If you'd met other requirements I'd have let this pass, but for the sake of meeting requirements of project, this should be name `main.js` as seen from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/dist_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page) It actually mentions in the project description to have webpack.config match the one from the link.

This is not as important as other requirements in terms of name but would be good to follow.

andynarf commented:
ok, since I didn't change an existing project, I wasn't as specific on the original PR .
## List of Changes : 
- added a new tab, changed the style to fit having an extra tab
- changed the filename to main.js from bundle.js 
- change id on html as requested
- made new js for each separated tab , and an content.js file ,that has the layout and skeleton of the page.
- added a form, since I thought it was fit for the contact tab.
